openam-authentication/wrenam-auth-duo/src/main/resources/amAuthDuo.properties

```diff
@@ -21,6 +21,7 @@ a502=Integration key
 a502.help=Cisco Duo application's integration key.
 a503=Secret key
 a503.help=Cisco Duo application's secret key.
+a504=Authentication timeout (ms)
```

Comment:
I don't think "authentication timeout" is the correct name for the options (it is a bit misleading). Maybe "check interval" or "poll interval"?

Reply:
See my comment below.